### DISCUSSION TITLE
How to get the function use for the plot

### DISCUSSION THREAD

**Discussion by JOYEXCENT** [Live-Charts/LiveCharts2#1616] [Q&A] [answered]
Hello,

Maybe someone knows the answer.

I want to get the function used by LiveCharts2 during the drawing of the curve in a CartesianChart. Any guidance or examples would be greatly appreciated!

Thank you!

**Comment by beto-rodriguez** [comment, marked as answer]
Hi

The source code of the LineSeries is here:

https://github.com/beto-rodriguez/LiveCharts2/blob/master/src/LiveChartsCore/CoreLineSeries.cs

**Reply by JOYEXCENT** [reply]
Hi, thanks I found what I search.
